src/entrypoints/scalars.ts

```diff
@@ -7,12 +8,13 @@ import { Json } from '../scalars/Json'
  *
  * | Prisma | GraphQL | Nexus `t` Helper | GraphQL Scalar Implementation |
```

Comment:
Let's add an "Additional Info" column and include MDN links like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt.